tests/test_filter_cellborn/results_true.dat

```diff
@@ -1,6 +1,6 @@
 k-combined:
 1.093844E+00 1.626801E-02
-tallies:
+tally 1:
```

Comment:
oh, it looks like it was added everywhere so I assume the general ``TestHarness`` class must insert it now. Disregard my question about it.